### ISSUE TITLE
Sprint Board View

### ISSUE THREAD

**Issue by jwthomp** [marc0l92/obsidian-jira-issue#21]
Is there a way to use the queries to generate a sprint board view where each story/bug is listed in a column based on status?
The table view is great, but there are times it is nice to see the sprint progress overview that the sprint board provides.

**Comment by marc0l92** [comment]
I agree that the sprint board may be interesting to view but I'm not sure what is the best way to reproduce it.

By default obsidian notes are 700px wide (except if you change it in the settings), and on mobile screens I have even less space.

If I create a kanban view like the jira sprint board I'm afraid it will not fit nicely in the note width.

In a basic jira workflow you have the status: open, in progress, in test, completed, accepted. With 5 status I will have around 140px per column that is not that much. 

Another option could be to follow the kanban plugin style and create a dedicated note that contain only the board. But I don't know how much it could be useful to have a dedicated note that looks like the board web page. The only advantage I see is that by clicking on the subtask you can directly jump to the dedicated note.

Another design option could be to display the issues in a table and show for each user story found with the jql query also the subtasks.

What was your idea behind this request? What is your use case?

**Comment by jwthomp** [comment]
What I was looking for was a way to easily view the status of a sprint, but where the stories are arranged by column state. This would allow me to not need to open up the JIRA app while working in Obsidian. If I could change the state of a story from within this view, that would be even better. 

I see what you are saying about the limited screen real estate. I'm currently using the kanban plugin for obsidian to manage my own work. I'm manually recreating the JIRA board locally, though I have to keep them in sync manually of course. One thing I'd like to be able to do for that is embed a JIRA issue link and status. However, to make this work, I'd need a way to not display the description of the task. So instead of formatting the jura-issue like: [BUG/STORYICON][STORYID][DESCRIPTION][STATUS], I would like to be able to limit it to [BUG/STORYICON][STORYID][STATUS] Perhaps something like this would work... (allow the user to specify which columns to display.
```jira-issue:icon,id,status
STORY-###
```

If you would like me to create a separate issue for that, let me know and I'd be happy to create one.

**Comment by marc0l92** [comment]
The creation of a nice Jira board will take some time.
In the meanwhile you can use the inline issue feature combined with the kanban plugin

![image](https://user-images.githubusercontent.com/12623073/181808594-d373117d-6b4b-4431-9709-f4e7c0e1e5af.png)

The issues will not be moved automatically in the correct column, but you will be able to see the status of them.

**Comment by JohnV7** [comment]
the combination with kanban is really useful. Is there a way to do the following:

1. create a column in Kanban
2. pull all jira tickets automatically, **not** manually
3. move **manually** in whatever column is needed
